python/py-mac-alias/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        dmgbuild mac_alias 2.2.2 v
+name                py-mac-alias
+revision            0
+
+supported_archs     noarch
+platforms           {darwin any}
+license             MIT
+
+maintainers         nomaintainer
+
+description         mac_alias lets you generate or read binary Alias and \
+                    Bookmark records from Python code.
+long_description    {*}${description}
+
+checksums           rmd160  683f3550d8b2f1f0b6732ea3f8d36c84ed249272 \
+                    sha256  175c8756d408593fcea6f3a86c31eb9d57505bb973d099f78ba33a9efff76f4f \
+                    size    31792
+
+python.versions     37 38 39 310 311 312
```

Comment:
please only add the Python subports you need (i.e., what is used for osxphotos) and certainly don't add EOL Python versions

Reply:
I just copied this line from another `py-*` Portfile. I thought it made sense to provide the module for all Python versions provided by MacPorts, but ok, I can limit it to just 3.12 if that’s the best practice.

Reply:
it's fine to add subports for older Python versions that are not EOL. However, if it's only needed now for Python 3.12 I would personally only add that one. That avoids the need to remove likely unused subports when other Python versions become EOL. If someone needs it, then it's always easy enough to add. 
So ling story short: if you want feel free to add others but not "37".